components/InterventionList.svelte

```diff
@@ -68,7 +72,7 @@
 <svelte:window on:keydown={onKeydown} />
 
 <Accordion>
-  {#each $gjScheme.features as feature, i}
+  {#each $gjScheme.features as feature, i (feature.id)}
```

Comment:
https://svelte.dev/tutorial/keyed-each-blocks
I can't tell if this was causing any actual problems earlier or not, but seems good to do this?

Reply:
Yes, it's looping over a list and enumerating the index. Spotty internet right now, but somewhere nearby in the Svelte tutorial it does that. What I added here was this "keyed each block" feature, which is supposed to help Svelte not rebuild the entire list when anything changes. (Or put things in the wrong order?) No behavior change that I could notice, but it still seems better